.azure-pipelines/templates/test-all-checks.yml

```diff
@@ -13,6 +13,9 @@ jobs:
     - checkName: datadog_checks_base
       displayName: Datadog Checks Base (Windows)
       os: windows
+      # Need to install Microsoft Visuall C++ 9.0 for python 2.7 manually because of
```

Comment:
Add also a note about which dependency is needed for so it pops in searches if we end up removing it